make/tools.mk

```diff
@@ -1,29 +1,17 @@
-CTR=docker
-
-# We don't need to re-download the whole Go distribution, but we still want
-# to make sure that we don't clutter the PATH by using a symlink. So we use
-# a symlink of the system Go in bin/tools.
-HELM=$(PWD)/bin/tools/helm
-COSIGN=$(PWD)/bin/tools/cosign
-CMREL=$(PWD)/bin/tools/cmrel
-YQ=$(PWD)/bin/tools/yq
-KUBECTL=$(PWD)/bin/tools/kubectl
-KIND=$(PWD)/bin/tools/kind
-CRANE=$(PWD)/bin/tools/crane
+# To make sure we use the right version of each tool, we put symlink in
+# bin/tools, and the actual binaries are in bin/downloaded. When bumping
+# the version of the tools, this symlink gets updated. The only limitation
+# is that moving from a version to another version that you previously
+# already had in cache won't work, and you will need to remove the
+# previously downloaded version first.
 
 # Let's have bin/tools in front of the PATH so that we don't inavertedly
 # pick up the wrong binary somewhere. Watch out, $(shell echo $$PATH) will
 # still print the original PATH, since GNU make does not honor exported
 # variables: https://stackoverflow.com/questions/54726457
 export PATH := $(PWD)/bin/tools:$(PATH)
 
-CLIENT_GEN=$(WORKDIR)/bin/tools/client-gen
-CONVERSION_GEN=$(WORKDIR)/bin/tools/conversion-gen
-CONTROLLER_GEN=$(WORKDIR)/bin/tools/controller-gen
-DEEPCOPY_GEN=$(WORKDIR)/bin/tools/deepcopy-gen
-DEFAULTER_GEN=$(WORKDIR)/bin/tools/defaulter-gen
-INFORMER_GEN=$(WORKDIR)/bin/tools/informer-gen
-LISTER_GEN=$(WORKDIR)/bin/tools/lister-gen
```

Comment:
These were never used anywhere.

Reply:
These were all added in preparation of porting https://github.com/cert-manager/cert-manager/blob/master/hack/verify-codegen.sh to make

Reply:
I propose to add them back when we do the porting, is that OK?